### DISCUSSION TITLE
Frequent Error terms

### DISCUSSION THREAD

**Discussion by Arasun01** [SYSTRAN/faster-whisper#458] [General]
While getting the transcript for the medical audio files, some of the medical terms were not transcribed correctly. I have 3.5k medical terms, How to add those lists to this faster whisper model?

Shall we apply some spell checker on the top of the transcripts produced by Faster Whisper? Anyone is facing this scenario? Any thought?

**Comment by etlweather** [comment]
I have a similar scenario, albeit a diffent corpus than medical. Currently I have a large file of "search and replace" that fixes some of the incorrectly transcribe subjet-matter terms. It's far from perfect as recording to recording, it does not make the same error each time. But it did help.

I am currently trying to do fine tuning to see if I can improve the recognition of the subject-matter terms. But that's going to take time as I have to have people go through and create the training corpus.

**Comment by guillaumekln** [comment]
You can try using `initial_prompt` to pass some of these medical terms (not all) which may help the model to use them in the generation. It's not guaranteed to work, however.

See also:

* https://github.com/openai/whisper/discussions/328
* https://github.com/openai/whisper/discussions/204

**Reply by abdulnim** [reply]
@guillaumekln what is the character or words limit for initial_prompt ?

**Reply by guillaumekln** [reply]
The limit is 224 tokens. If there are more tokens they are simply ignored.

**Reply by toanhn** [reply]
@guillaumekln Does intial_promp affect the accuracy of transcription?

**Comment by toanhuynhnguyen** [comment]
@guillaumekln  The "initial_prompt" sometimes cause missing some sentences. How can I solve this issue?
